compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -286,6 +293,9 @@ impl<'cx, 'tcx> crate::MirBorrowckCtxt<'cx, 'tcx> {
             (&ty::Slice(_), _) => "slice",
             _ => span_bug!(move_from_span, "this path should not cause illegal move"),
         };
+        //FIXME: failed ui test diag-migration
+        //       -    error[E0508]: cannot move out of type `[S; 1]`, a non-copy array
+        //       +    error[E0508]: cannot move out of type `S`, a non-copy array
```

Comment:
It's difficult to say what was the problem here without seeing the code, this error would suggest just a mistake in the migration.